doc/userguide/upgrade.rst

```diff
@@ -71,6 +71,8 @@ Major changes
 - PF_RING support has been moved to a plugin. See :doc:`PF_RING plugin
   <upgrade/8.0-pfring-plugin>`.
 - LDAP parser and logger have been introduced.
+- NAPATECH support has been moved to a plugin. See :doc:`NAPATECH plugin
+  <upgrade/8.0-napatech-plugin>`.
```

Comment:
Does NAPATECH refer to themselves in all caps?

Reply:
They do not; i'll update the references.